portfolio/src/main/webapp/style.css

```diff
@@ -1,129 +1,92 @@
-input {
-  border-bottom: solid 1px;
-  border-left: none;
-  border-right: none;
-  border-top: none;
-  height: 60px;
-  outline: none;
-  width: 50%;
-}
-
-.form-button {
-  height: 60px;
-  margin-bottom: 10px;
-  margin-top: 10px;
-  width: 50%;
-}
-
-.about-container .description p {
-  margin-left: 30px;
-}
-
-.work-container {
+.footer-container {
   display: flex;
   flex-direction: column;
-  justify-content: space-around;
-  margin-bottom: 10px;
-  margin-top: 100px;
-  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1000px;
 }
-
-.jobs-container {
+.contact-form {
+  align-items: center;
   display: flex;
   flex-direction: column;
-}
-
-.jobs-container .work {
+  height: 100%;
   margin-left: auto;
   margin-right: auto;
+  width: 100%;
 }
-
-.work img {
-  max-width: 500px;
-  min-height: 300px;
-  min-width: 200px;
-}
-
-.projects-container {
+.comments-container {
+  align-items: center;
   display: flex;
-  justify-content: space-around;
-}
-
-.projects-container div {
-  width: 30%;
-}
-
-.about-container img {
-  height: auto;
-  max-width: 500px;
-  min-height: 300px;
-  min-width: 200px;
-  width: 100%;
+  flex-direction: column;
 }
-
-img {
-  height: auto;
-  min-height: 300px;
-  min-width: 300px;
+.menu-bar {
+  align-items: center;
+  background-color: rgb(51,51,51);
+  display: flex;
+  height: 45px;
+  justify-content: center;
   width: 100%;
 }
-
-.header-container {
+.menu-content {
   align-items: center;
+  color: white;
   display: flex;
-  height: 100px;
-  justify-content: space-evenly;
-  margin: 0 auto 10px auto;
-  max-width: 1000px;
-  width: 100%;
+  height: auto;
+  justify-content: center;
+  width: auto;
 }
-
-.header-item {
-  color: black;
-  font-size: 23px;
-  text-decoration: none;
+.menu-item {
+  font-size: 30px;
+  margin-bottom: auto;
+  margin-top: auto;
```

Comment:
This is because you're using a `<p>` tag which has a bottom margin. Consider using a `<div>` in this case which doesn't have this margin or removing the bottom margin with `margin-bottom: 0;`. 
![image](https://user-images.githubusercontent.com/8546611/84402722-d26c0d80-abd2-11ea-87f4-3a3548d21c6e.png)